src/app/layout/components/user-menu/user-did/user-did.component.html

```diff
@@ -1,5 +1,9 @@
-<div class="d-flex align-items-center justify-content-center did-wrapper mx-2 px-3 mb-2"
-     appCopyToClipboard [copyClipboard]="did" message="User Did">
-  <div class="f-14" data-qa-id="menu-did">{{did | didFormatMinifier}}</div>
+<div
+  class="d-flex align-items-center justify-content-center did-wrapper mx-2 px-3 mb-2"
+  appCopyToClipboard
+  [copyClipboard]="did"
+  message="User Did"
+>
```

Comment:
I really do not like this change. Why it can't be like this?
```
<div class="d-flex align-items-center justify-content-center did-wrapper mx-2 px-3 mb-2"
     appCopyToClipboard
     [copyClipboard]="did"
     message="User Did">
```
Switchboard is the only frontend/angular app that seems to need use this, why we do not adjust it? If not in general then only in this app?